src/crimson/os/seastore/extent_placement_manager.h

```diff
@@ -326,12 +328,14 @@ class ExtentPlacementManager {
   TCachedExtentRef<T> alloc_new_extent(
     Transaction& t,
     segment_off_t length,
-    placement_hint_t hint = placement_hint_t::NONE) {
+    placement_hint_t hint) {
     // only logical extents should fall in this path
     static_assert(is_logical_type(T::TYPE));
     assert(hint < placement_hint_t::NUM_HINTS);
     auto dtype = get_allocator_type(hint);
-    bool delay = can_delay_allocation(dtype);
+    // FIXME: set delay for COLD extent when the record overhead is low
+    bool delay = (hint > placement_hint_t::COLD &&
```

Comment:
could you please check the comments and the condition check?  For COLD extent, hint will be  placement_hint_t::COLD, right? then delay is false here.

Reply:
Refer to the reply below, we probably want to do ool writes for COLD extents with larger RGW writes.